### DISCUSSION TITLE
Can this workbench work with FreeCAD 1.0 Assembly?

### DISCUSSION THREAD

**Discussion by chen-tianjian** [drfenixion/freecad.robotcad#56] [Q&A] [answered]
Does it support importing and exporting URDF from the latest FreeCAD 1.0 official Assembly workbench https://wiki.freecad.org/Assembly_Workbench ?
If not yet, do you have any plans to do so?
Thanks!

**Comment by drfenixion** [comment, marked as answer]
Hello. RobotCAD is some type of assembly workbench itself. Integrating support of official Assemble inside RobotCAD looks like big task. I dont have any plan to do that currently.

**Comment by drfenixion** [comment]
You can also ask in upstream repo - "CROSS". If they made i will integrate this feature.
